docs/HOWTO/mri_reconstruction_demo.py

```diff
@@ -6,7 +6,7 @@
 import cv2 as cv
 import argparse
 import time
-from openvino.inference_engine import IECore
```

Comment:
this looks like a really old file, because we don't expose old API (inference_engine) in our documentation
maybe we need to completely remove it? if we don't have any references to it

Reply:
We do not have any references to it apparently, so I will remove it.